includes/Core/Assets/Manifest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Assets\Manifest
+ *
+ * @package   GoogleSite_Kit
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Assets;
+
+use Google\Site_Kit\Plugin;
+
+/**
+ * Assets manifest.
+ *
+ * @since 1.15.0
+ * @access private
+ * @ignore
+ */
+class Manifest {
+
+	/**
+	 * Files as $handle => $filename map.
+	 *
+	 * @since n.e.x.t
+	 * @var array|null
+	 */
+	private static $assets;
+
+	/**
+	 * Gets the filename for a given handle.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $handle  Script or stylesheet handle.
+	 */
+	public static function get_filename($handle ) {
```

Comment:
Odd that PHPCS didn't flag this 🤷‍♂️ 

```suggestion
	public static function get_filename( $handle ) {
```